tests/LoggingTest.php

```diff
@@ -73,6 +71,38 @@ public function testBuildTagNotIncluded()
         $this->assertArrayNotHasKey('build_tag', $records[0]['extra']);
     }
 
+    public function testUrlPatternIncluded()
+    {
+        // configure a test logger
+        $handler = new TestHandler();
+        $logger = new Logger('testing', [$handler]);
+        $logger->pushProcessor(new UrlPatternProcessor());
+        Log::swap($logger);
+
+        // log something
+        Log::info('testing');
+
+        // then assert that the build tag is present
+        $records = $handler->getRecords();
+        $this->assertEquals('', $records[0]['extra']['url_pattern']);
+    }
+
+    public function testUrlPatternNotIncluded()
```

Comment:
The setup in these two tests is the same but there are different expectations (one of which is failing).

Reply:
Yeah sorry I haven't quite finished yet. This test is correct but the above one needs updating.  
I need to figure out how to mock the current request to assert that the log field is actually added to the log record